### PR TITLE
regressions found while fixing tests

### PR DESCRIPTION
**Pull request by wargio**
REF: https://github.com/radare/radare2-regressions/pull/765

### CODE REVIEW

**Comment by radare** [comment]
eta wen

**Comment by radare** [comment]
what about merging this first and then doing another pr for the pending sparc issues?

File: libr/parse/parse.c [hunk 1]
```diff
@@ -139,7 +139,17 @@ static char *findNextNumber(char *op) {
 				bool is_space = ansi_found;
 				ansi_found = false;
 				if (!is_space) {
-					is_space = (p == op || *o == ' ' || *o == ',' || *o == '[');
+					is_space = (p == op || *o == ' ' || *o == ',');
+				}
+				if (*p == '[') {
+					char *t = p;
+					p++;
+					if (!IS_DIGIT(*p)) {
```
Comment (radare) [inline libr/parse/parse.c]: missing space before (

File: libr/parse/parse.c [hunk 2]
```diff
@@ -139,7 +139,17 @@ static char *findNextNumber(char *op) {
...
+						for (;*p && *p != ']'; p++);
+						if (*p == ']') {
+							continue;
+						} else p = t;
```
Comment (radare) [inline libr/parse/parse.c]: missing braces

**Comment by wargio** [comment]
i agree on merging this, then add a new issue with those `sparc` related issues.
i'll do the same on the r2r, ok?

**Comment by radare** [comment]
ok! fix the comments and ill merge this.

Thanks!

**Comment by wargio** [comment]
Ok, should be ok now to merge. let's see what travis says :)

**Pull request merged**